README.md

```diff
@@ -0,0 +1,94 @@
+# Managed service controller
+
+Kubernetes Controller to create a managed service namespace.
+
+A managed service namespace will consist of managed services.
```

Comment:
minor but prob should be single tenant managed services shared by a single tenant across multiple consumer namespaces.
Something like AMQ may go in here for instance and be shared across multiple consumer namespaces

Reply:
Does this cover it?

"A managed service namespace will consist of managed services.
A managed service is something that a user of a cluster can make use of without needing to be concerned about the maintenance and overhead of operating that service.

These are usually heavy weight services that will **be single tenant services shared by a single tenant across multiple consumer namespaces.**"